<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>631</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>901</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>정보입력</string>
       </property>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>400</width>
          <height>30</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhLatinOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>40</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>검색 : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btnShowAns">
        <property name="geometry">
         <rect>
          <x>742</x>
          <y>20</y>
          <width>69</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>정답보기</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnHideAns">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>20</y>
          <width>73</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>정답 가리기</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSearch">
        <property name="geometry">
         <rect>
          <x>547</x>
          <y>20</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>검색</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpenPro">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>73</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>문제 출력</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRandPro">
        <property name="geometry">
         <rect>
          <x>627</x>
          <y>20</y>
          <width>109</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>문제랜덤 출력</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>901</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tblcont"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stbCurrent"/>
 </widget>
 <resources/>
 <connections/>
</ui>
